fearless/Common/Services/ExtrinsicService/ExtrinsicService.swift

```diff
@@ -24,12 +94,27 @@ protocol ExtrinsicServiceProtocol {
         completion completionClosure: @escaping EstimateFeeClosure
     )
 
+    func estimateFee(
+        _ closure: @escaping ExtrinsicBuilderIndexedClosure,
+        runningIn queue: DispatchQueue,
+        numberOfExtrinsics: Int,
+        completion completionClosure: @escaping EstimateFeeIndexedClosure
+    )
+
     func submit(
         _ closure: @escaping ExtrinsicBuilderClosure,
         signer: SigningWrapperProtocol,
         runningIn queue: DispatchQueue,
         completion completionClosure: @escaping ExtrinsicSubmitClosure
```

Comment:
Please, remove. We don't need this anymore

Reply:
Do you suggest to create wrappers instead?

Reply:
We have this in extension, so we don't need to request implementation of it in protocol